lib/user.js

```diff
@@ -74,13 +82,15 @@ const insertNewUser = async function (claims) {
 const ensureFreshUserData = async function (person, claims) {
   const properties = [
     { predicate: 'foaf:firstName', oldValue: person.firstName, newValue: claims[AUTH_FIRST_NAME_CLAIM] },
-    { predicate: 'foaf:familyName', oldValue: person.familyName, newValue: claims[AUTH_FAMILY_NAME_CLAIM] }
+    { predicate: 'foaf:familyName', oldValue: person.familyName, newValue: claims[AUTH_FAMILY_NAME_CLAIM] },
+    { predicate: 'adms:status', newValue: ACCESS_ALLOWED_STATUS_URI },
```

Comment:
Waarom is dit nodig als je de status zijn in de `INSERT DATA`? Ga je hiermee de status niet ongewild overschrijven?

Reply:
Can be removed @sergiofenoll ?